### ISSUE TITLE
Pass parameters with \require{}

### ISSUE THREAD

**Issue by mhchem** [mathjax/MathJax#1708]
Is it possible to pass parameters to a `\require{}` command, like `\require[legacy=true]{mhchem}`? If yes: I don't find it in the documentation. If no: Should it be added?

**Comment by pkra** [comment]
> Is it possible to pass parameters to a \require{} command,

I don't think it is, cf. https://github.com/mathjax/MathJax/blob/master/unpacked/jax/input/TeX/jax.js#L1629-L1634

**Comment by dpvc** [comment]
Peter is correct, there is no current way to do this.  I will think about how it could be managed, and what the various use cases might be.
